guides/elements.md

```diff
@@ -480,3 +531,25 @@ de:
       contact_form:
         color: Button Farbe
 ~~~
+
+## Notes on interacting with Elements from the Admin interface
+
+While this guide is intended for developers building a CMS site, many developer will need to train clients and customers, or interact with the admin interface themselves.
+
+While the Admin UI tries to be as intuitive as possible, there are a few functions that aren't quite as obvious.
+
+### Re-arranging Elements on a Page
+
+Collapsed elements can be re-arranged by clicking and dragging the window icon to the left of the element title.
+
+You can't re-arrange an expanded element, you need to collapse it first.
```

Comment:
This is not true. Elements can be re-arranged all the time. There is no such feature. Please remove this paragraph

Reply:
I just tested on several pages with elements of various complexity. When an element is un-collapsed, it doesn't register correctly with the jQuery UI and can't be re-ordered.

Reply:
This is a bug then. Can you please open an issue in the Alchemy repo and provide reproduction steps on a vanilla Alchemy install? Thanks

Reply:
This PR is getting a bit cumbersome, I'll just delete the note. I used the guide to do a vanilla alchemy install, which wasn't easy given the guide seems to live somewhere around v4.4. I also don't think this PR solves the real problem which is the lack of an actual v6 guide. I think we're both putting more time into this PR than it's worth.

What I really want to contribute to is an actual v6 guide that talks about ingredients from the get-go. This PR feels like a back-port for v4.4 that no one will actually use if they are starting fresh with Alchemy and need the guides.

Reply:
You are right, thanks for the patience, but the time is not wasted, since we will start the v6 guide based on this work.